docs/monitors/elasticsearch.md

```diff
@@ -11,77 +11,116 @@ This monitor collects cluster level and index level stats only from the current
 in an Elasticsearch cluster by default. It is possible to override this with the
 `clusterHealthStatsMasterOnly` and `indexStatsMasterOnly` config options respectively.
 
-A simple configuration that collects only default metrics looks like the following
+A simple configuration that collects only default, included metrics looks like the following:
```

Comment:
It looks like there may be two spaces after the comma?

Reply:
I checked, it's only one.